src/pages/Home/About/RoadMapSection/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from "react";
+import { YEARS } from "./RoadMapContent";
+import DisplayEvents from "@/components/DisplayEvents";
+import makeArc from "@/helpers/makeArc";
+import displayQuarterlyEvent from "@/helpers/displayQuarterlyEvent";
+import ArchLine from "@/assets/icons/arch_line.svg";
+import styles from "./RoadMapSection.module.scss";
+
+const FIRST = ["Jan", "Feb", "Mar"];
+const SECOND = ["Apr", "May", "Jun"];
+const THIRD = ["Jul", "Aug", "Sep"];
+const FOURTH = ["Oct", "Nov", "Dec"];
+
+const YEARS_ARRAY = [2017, 2018, 2019, 2020, 2021, 2022, 2023];
+
+function RoadMap() {
+  const current = new Date();
+  const currentYear = current.getFullYear();
+  const [yearSelected, setYearSelected] = useState(currentYear);
+  const getQuarterFour = displayQuarterlyEvent(YEARS[yearSelected], FOURTH);
+  const getQuarterThree = displayQuarterlyEvent(YEARS[yearSelected], THIRD);
+  const getQuarterTwo = displayQuarterlyEvent(YEARS[yearSelected], SECOND);
+  const getQuarterOne = displayQuarterlyEvent(YEARS[yearSelected], FIRST);
+
+  const getArch = makeArc(YEARS_ARRAY);
```

Comment:
Is this actually still being used, or is that a later pull request

Reply:
oh yes, actually we have decided to keep it for now cos it looks good on the desktop and tablet screen size. Andrea will design one for the mobile size. What do you think?